src/components/Calender.vue

```diff
@@ -15,7 +15,15 @@
 		>
 			<template v-if="!calVal.isPadding">
 				<h3>
-					{{ calVal.date + 1 }}
+					{{
+						"".concat(
```

Comment:
You don't need to concatenate the string in JS, you can just let the two strings be combined in the html. like:
```
<h3>
    {{ calVal.date + 1 }}
    {{ filmExist(calVal.date)) ? listToDisplay.get(calVal.date)!.name : '' }}
</h3>
```

Reply:
Oooh is that a vue3 thing?

Reply:
Not really, same idea would work in vue2/react/etc. More of a trick exploiting the fact that HTML trims starting and trailing whitespace, and also collapses all whitespace between text into a single space. (Exception: the `<pre>` tag preserves all whitespace, and is used mostly for code blocks)